lib/project.go

```diff
@@ -22,6 +22,11 @@ type Project struct {
 	BrotherProject *Project `json:"-"`
 }
 
+type ProjectJSON struct {
+	ID   int    `json:"ID"`
+	Name string `json:"Name"`
+}
```

Comment:
The `lib` directory (package) should focus on Todoist API.
So I don't want to place this code here.
This should be placed on './projects.go' .